edgePy/data/groups.txt

```diff
@@ -0,0 +1,2 @@
+Group 1: A_1, A_2, A_3, A_4, A_5
+Group 2: B_1, B_2, B_3, B_4, B_5
```

Comment:
Is this a common file format for defining the groups?

Is this TSV or whitespace delimited?

Also, if this is used only for testing it does not need to be a physical file. Like so:

Reply:
I'm not aware of a common format for groups - am totally open to one, however.  We need to decide on it. For now, it's:
Group:sample,sample2,sample3

Group name, colon, sample [comma, sample]